manifests/kustomize/env/cert-manager/platform-agnostic-multi-user-k8s-native/kustomization.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../platform-agnostic-k8s-native
+  - ../../../third-party/metacontroller/base
+  - ../../../base/installs/multi-user
+  - ../../../base/metadata/options/istio
+  - ../../../third-party/argo/installs/cluster
+  - ../../../third-party/mysql/options/istio
+  - ../../../third-party/minio/options/istio
+
+# Identifier for application manager to apply ownerReference.
+# The ownerReference ensures the resources get garbage collected
+# when application is deleted.
+commonLabels:
+  application-crd-id: kubeflow-pipelines
+
+# !!! If you want to customize the namespace,
+# please also update base/cache-deployer/cluster-scoped/cache-deployer-clusterrolebinding.yaml
+namespace: kubeflow
+
+patches:
+  - patch: |-
+      - op: add
```

Comment:
Could we move these patches to a separate file so that both new kustomization.yaml files can leverage it without duplicating? See the doc here that shows how to do this:
https://github.com/kubernetes-sigs/kustomize/blob/master/examples/jsonpatch.md